tests/utils/pod.go

```diff
@@ -79,6 +81,14 @@ func NewPodWithPVC(podName, cmd string, pvc *k8sv1.PersistentVolumeClaim) *k8sv1
 					Name:    "runner",
 					Image:   "kubevirt/cdi-importer:latest",
 					Command: []string{"/bin/sh", "-c", cmd},
+					Resources: k8sv1.ResourceRequirements{
```

Comment:
This doesn't consult CDIConfig.  Is that going to be a problem?

Reply:
This is for verifier pods and things of that nature, its not actually part of CDI.